libcontainer/cgroups/fs/memory.go

```diff
@@ -16,10 +19,19 @@ import (
 )
 
 const (
-	cgroupMemorySwapLimit = "memory.memsw.limit_in_bytes"
-	cgroupMemoryLimit     = "memory.limit_in_bytes"
+	numaStatColumnSeparator   = " "
+	numaStatKeyValueSeparator = "="
+	numaStatMaxColumns        = math.MaxUint8 + 1
+	numaStatFileName          = "memory.numa_stat"
```

Comment:
It would be great to keep consistency with these consts created below:
```
cgroupMemorySwapLimit     = "memory.memsw.limit_in_bytes"
cgroupMemoryLimit         = "memory.limit_in_bytes"
```